app/src/main/res/values/strings.xml

```diff
@@ -184,21 +185,21 @@
     <string name="importVoucherVaultMessage">Select your <i>vouchervault.json</i> export from Voucher Vault to import.
 \nCreate it by pressing Export in Voucher Vault first.</string>
     <string name="barcodeId">Barcode value</string>
-    <string name="sameAsCardId">Same as card ID</string>
+    <string name="sameAsCardId">Same as ID</string>
     <string name="setBarcodeId">Set barcode value</string>
     <string name="unsupportedBarcodeType">This barcode type can\'t yet be displayed. It may be supported in a later version of the app.</string>
     <string name="wrongValueForBarcodeType">The value is not valid for the selected barcode type</string>
-    <string name="copy_to_clipboard_multiple_toast">Card IDs copied to clipboard</string>
-    <string name="intent_import_card_from_url_share_multiple_text">I want to share some cards with you</string>
-    <string name="frontImageDescription">Card\'s front image</string>
-    <string name="backImageDescription">Card\'s back image</string>
+    <string name="copy_to_clipboard_multiple_toast">IDs copied to clipboard</string>
+    <string name="intent_import_card_from_url_share_multiple_text">I want to share something from Catima with you</string>
```

Comment:
This is shown to people who may not have the app. "Something from Catima" is super unclear. Like, a card? A movie? A banana?